Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1427,6 +1401,24 @@ def audit_urls
       next unless u =~ %r{https?://(?:central|repo\d+)\.maven\.org/maven2/(.+)$}
       problem "#{u} should be `https://search.maven.org/remotecontent?filepath=#{$1}`"
     end
+
+    return unless @online
+    urls.each do |url|
+      if url.start_with? "http", "ftp"
+        status_code = url_status_code url
+        unless status_code.start_with? "2"
```

Comment:
An example which returns a non `20x` code is `brew audit --online rc` which returns 226.